cisco_sdwan/assets/monitors/tunnel_down.json

```diff
@@ -0,0 +1,31 @@
+{
+	"version": 2,
+	"created_at": "2024-04-05",
+	"last_updated_at": "2024-04-05",
+	"title": "Cisco SD-WAN Tunnel is down",
+	"tags": [
+		"integration:cisco-sdwan"
+	],
+	"description": "This monitor detects when a Cisco SD-WAN tunnel becomes unavailable between two devices. It tracks the status of tunnels within a specified namespace and triggers an alert if a tunnel goes down, indicating potential connectivity issues.",
```

Comment:
Should it be `between two sites` or `between two devices`?

Reply:
`between two devices` is more correct as you can have several edge devices in one site (e.g. for redundancy)

Reply:
👍